products/sle/main.pm

```diff
@@ -27,9 +27,12 @@ BEGIN {
 }
 use utils;
 use main_common;
+use known_bugs;
 
 init_main();
 
+set_serial_failures;
```

Comment:
I would add some comment, so that people know what's it about, or even better rename the function to `set known error detection` or something similar.